install.sh

```diff
@@ -3,6 +3,8 @@
 # Select browser
 if [[ -n $1 ]]; then
   browser=$1
+elif [[ -n $BROWSER ]]; then
+  browser=$(basename "$BROWSER")
```

Comment:
Extreme nitpick: unambiguous expansions:

```suggestion
elif [[ -n "${BROWSER}"]]; then
  browser=$(basename "${BROWSER}")
```

Reply:
I applied that. You missed the space before the `]]` though